npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -90,9 +90,8 @@ func (pMgr *PolicyManager) initializeNPMChains() error {
 // removeNPMChains removes the jump rule from FORWARD chain to AZURE-NPM chain
 // and flushes and deletes all NPM Chains.
 func (pMgr *PolicyManager) removeNPMChains() error {
-	deleteErrCode, deleteErr := pMgr.runIPTablesCommand(util.IptablesDeletionFlag, jumpFromForwardToAzureChainArgs...)
-	hadDeleteError := deleteErr != nil && deleteErrCode != couldntLoadTargetErrorCode
-	if hadDeleteError {
+	deleteErrCode, deleteErr := pMgr.runIPTablesCommandAndIgnoreErrorCode(couldntLoadTargetErrorCode, util.IptablesDeletionFlag, jumpFromForwardToAzureChainArgs...)
```

Comment:
In deletion operation, is it ok not to get error code about `doesNotExistErrorCode` or `couldntLoadTargetErrorCode`, etc?
Anyway, the goal is to clean up the tables.

Reply:
if the AZURE-NPM chain doesn't exist, we'll get couldntLoadTargetErrorCode and if the chain exists but the rule doesn't exist, we'll get doesNotExistErrorCode. So this should ignore both of these codes actually, since this is called before initializing DP.

Reply:
if there's an error besides those two, something's wrong, but it could be several things, so ya maybe we just log and continue.

For instance, we get `couldntLoadTargetErrorCode` for "unknown option" and for a malformed IP address.

Reply:
these changes will be moved to the next PR which updates policy/chain management